.github/workflows/check-migrations.yml

```diff
@@ -0,0 +1,35 @@
+name: Check Migrations on Paseo
+
+on:
+  pull_request_target:
+    types:
+      - labeled
```

Comment:
Triggers the workflow when a PR is labeled

Reply:
Below we check for two conditions

    `if: github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'change/storage-migration')`


So lable should be `change/storage-migration` for the job to be triggered